### DISCUSSION TITLE
How to center the text?

### DISCUSSION THREAD

**Discussion by zSynctic** [flet-dev/flet#721] [Q&A] [answered]
This is my code: 
import flet
from flet import IconButton, Page, Row, TextField, icons

def main(page: Page):
    page.title = "test page"
    page.vertical_alignment = "center"
    page.horizontal_alignment = "center"

    txt = TextField(value="hi", text_size=20, text_align=flet.TextAlign.CENTER, width=200, height=50, autofocus=True)

    page.add(
        flet.Row(
            [
                txt
            ],
            alignment=flet.MainAxisAlignment.CENTER
        )
    )

 flet.app(target=main)

but the text in the textfield isn't in the center even when aligning it
![image](https://user-images.githubusercontent.com/71632495/208264429-b86c1f30-cc26-4c92-b2ae-7456820a0386.png)

**Comment by Francois-Belanger** [comment, marked as answer]
Try removing the "height=50"

[Image.pdf](https://github.com/flet-dev/flet/files/10252738/Image.pdf)

**Reply by zSynctic** [reply]
Thanks it worked
